### ISSUE TITLE
Document supported OS/web browsers for clients

### ISSUE THREAD

**Issue by TheFrenchGhosty** [iv-org/documentation#456]
Originally discussed on Marix: 
![alt text bellow](https://github.com/iv-org/documentation/assets/47571719/b18f445d-dc20-4552-b90f-0c13c0922483)

Text:

TheFrenchGhosty
Anyway, on the subject of Internet Explorer and browser supportunixfox Samantaz Fox we should work on a list of browser that have to work even partially, and maybe even a matrix to show which feature don't work on X browser
For example, Firefox and Chrom* must be supportes, followed by Palemoon and the exotic forks of Firefox and Chrom* (Thorium, LibreWolf...), followed maybe by New Moon https://o.rthost.win/palemoon/
unixfox
good idea, do you want to contribute to this list 😆😁?
(create a github issue if you can)
TheFrenchGhosty
We have to agree on the list :3 
Personally I want us to support any 32bit Linux distro + From windows XP 32bit to *current windows*

![alt text bellow](https://github.com/iv-org/documentation/assets/47571719/5302e972-5ee8-40ce-ae1d-12856d87568e)

Text: 

TheFrenchGhosty
I was talking about client side support when I mentioned OS
Basically, we should support a good FOSS browser that's available on those OS

@Ryuno-Ki  recommended http://browserl.ist/ / https://github.com/mdn/browser-compat-data
